readhomer_atlas/library/models.py

```diff
@@ -396,14 +419,14 @@ def tokenize(cls, text_part_node, counters):
             subref_idx = idx[w]
             subref_value = f"{w}[{subref_idx}]"
 
+            position = pos + 1
             to_create.append(
                 cls(
                     text_part=text_part_node,
                     value=piece,
                     word_value=w,
-                    position=pos + 1,
-                    # @@@ not a true uuid
-                    uuid=f"t{text_part_node.ref}_{pos}",
+                    position=position,
```

Comment:
@jtauber: What do you think about the concept of a `ve_ref` (vs what is called a `uuid` in the current data set)?

The first token of 1.1 would be `1.1.t1`, etc.

A true "tokenized" exemplar would be `1.1.1`, but I haven't wanted to introduce that type of exemplar into ATLAS just yet.

Basically, I want a value that we can use on a token without having to do a join for the token's text part.  If you'd prefer a different syntax for the value, or have a preferred name over `ve_ref`, I'm quite open to it.